src/validation_utils.c

```diff
@@ -61,6 +64,10 @@ char	*get_executable_path(char **paths, char *executable)
 		free(full_path);
 		i++;
 	}
-	ft_putstr_fd("Executable not found\n", 2);
+	g_exit = 127;
+	msg = ft_strjoin(executable, ": command not found\n", NULL);
```

Comment:
Dixt, I don't know how yet, but I want to use errno fully so we don't need to hardcode any basic message.

Reply:
errno was returning different messages compared to bash (I can re-test and show the differences). I hardcoded this part just to have exactly the same message as bash.